app/models/Campaign.coffee

```diff
@@ -40,6 +40,11 @@ module.exports = class Campaign extends CocoModel
       levels.push({key: level.original, practice, assessment})
     return utils.createLevelNumberMap(levels)
 
+  getLevelNameMap: () ->
```

Comment:
Should we cache this map and only recreate when necessary?
Seems like *maybe* we only call this once currently but may be a good practice to go ahead and do as necessary.

Reply:
Yeah, thats a good point. Seems like we are using `lscache` for caching such things.